crates/provider/src/docker/node.rs

```diff
@@ -501,9 +501,30 @@ where
 
     async fn run_script(
         &self,
-        _options: RunScriptOptions,
+        options: RunScriptOptions,
     ) -> Result<ExecutionResult, ProviderError> {
-        todo!()
+        let file_name = options
+            .local_script_path
+            .file_name()
+            .expect(&format!(
+                "file name should be present at this point {THIS_IS_A_BUG}"
+            ))
+            .to_string_lossy();
+
+        let remote_script_path = PathBuf::from(format!("/tmp/{file_name}"));
```

Comment:
Is better to use the `scripts_dir` from the node (_provider node_) itself. (https://github.com/paritytech/zombienet-sdk/blob/main/crates/provider/src/docker/node.rs#L104)